README.txt

```diff
@@ -8,45 +8,74 @@ README.txt
 
 alternatebitprotocol.doc
 
-atomics [This folder contains the header files]
-	receiverCadmium.hpp
-	senderCadmium.hpp
-	subnetCadmium.hpp
+bin [This folder contains all .exe files generated for the project]
+	test [This folder contains all .exe files generated for testing only]
+		RECEIVER_TEST.exe
+		SENDER_TEST.exe
+		SUBNET_TEST.exe
+	top_model [This folder contains all .exe files generated for simulator only]
+	ABP.exe

```

Comment:
ABP.exe should be indented (move right one tab or 4 spaces)

Reply:
fixed it. please review the latest README file